main/station-config.js

```diff
@@ -85,16 +89,65 @@ function setFilAddress (address) {
 /**
  * @returns {string}
  */
-function getStationID () {
+function getStationID() {
   return StationID
 }
 
+/**
+ * @returns {string}
+ */
+function getStationWalletAddress() {
+  return FilAddress || '' // needs refactor
```

Comment:
which refactor does this need?

Reply:
Sorry, this comment should have been removed.
For context, I was stimulating the address for ad-hoc testing like this
`return FilAddress || 'f0111' // needs refactor`